lib/msf/core/handler/reverse_http.rb

```diff
@@ -356,6 +356,23 @@ def on_request(cli, req)
 
           print_status("Staging #{uuid.arch} payload (#{blob.length} bytes) ...")
 
+
+          # Checking whether LHOST is a loopback address
+
+          def is_loopback_address(address)
```

Comment:
Remove this, since it'll be available in the included `Msf::Handler::Reverse` once you move it there.